sympy/polys/matrices/domainmatrix.py

```diff
@@ -1749,3 +1739,43 @@ def lll(A, delta=QQ(3, 4)) -> 'DomainMatrix':
 
         """
         return DomainMatrix.from_rep(A.rep.lll(delta=delta))
+
+    def lll_transform(A, delta=QQ(3, 4)):
+        """
+        Please see the docstring for lll. This method returns the reduced basis
```

Comment:
It would be better to have a clickable reference for the html docs by using a sphinx reference. We can also add a "See Also" section at the bottom.

I would probably write this as
```
LLL-reduced basis and transformation matrix.

Examples
========

>>> from sympy import ZZ
>>> from sympy.polys.matrices import DM
>>> A = DM([[1, 2], [3, 4]], ZZ)
>>> B, T = A.lll_transform()
>>> T*X == B
True

Explanation
===========

Parameters, algorithm and basis are the same as for :meth:`lll` except that the
return value is a tuple `(B, T)` with `B` the reduced basis and `T` a
transformation matrix. The original basis `A` is transformed to `B` with
`T*A == B`. If only `B` is needed then :meth:`lll` should be used as it is a
little faster.

See also
========

lll
```

Reply:
> It would be better to have a clickable reference for the html docs by using a sphinx reference.
Thanks for noticing this, I meant to ask about that. Do you know about how to do this in a docstring? I found some documentation for referencing another part of the docs, but I don't understand how I should do that inside a docstring. Thanks.

Reply:
You can use something like
```
:meth:`lll`
```
to create a clickable link to the `lll` method:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html
https://docs.sympy.org/latest/contributing/docstring.html#cross-referencing

Also anything in the "See also" will be clickable but that uses a different syntax that I have never fully pinned down.